cpp/models/abm/location.cpp

```diff
@@ -129,12 +128,12 @@ void Location::add_person(const Person& p)
     if (!m_cells.empty()) {
         for (auto i : p.get_cells()) {
             ++m_cells[i].num_people;
```

Comment:
We should again discuss the usage of cells. As the current implementation would mean that you double your transmission risk if you were in two cells, don't you?

Reply:
This is also in issue I discussed with Carlotta. Yes, it does, so I agree that we should discuss what cells are for and where they should be used.

Reply:
Usage of cells should be discussed separately from this issue.